app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,11 @@
+class ReviewsController<ApplicationController
+  before_action :set_user
+
+  def set_user
+    @user = User.find(params[:user_id])
```

Comment:
I am curious to hear more about why we need #set_user in this controller and the users_controller.

Reply:
Honestly, because I tried calling it in both and it only works if you define it in the controller you're using it in

Reply:
Gotcha! Thanks for the explanation.